Makefile

```diff
@@ -9,16 +9,17 @@ size  :; forge build --sizes
 inspect :; forge inspect ${contract} storage-layout --pretty
 
 FORK_URL := ${ETH_RPC_URL} 
+FORK_BLOCK_NUMBER := 19435506
```

Comment:
why are we fixing a block number?

Reply:
It's faster to run the tests when you are on a fixed number. Lower bandwidth. I'll remove it.

Reply:
https://github.com/newmickymousse/yeth-strategy/pull/2/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R14